tests/box2d/Benchmark.cpp

```diff
@@ -29,6 +27,10 @@ typedef struct {
 #include <emscripten.h>
 #endif
 
+#if NO_PRINTING
+#define printf(fmt, ...) (0)
```

Comment:
I assume this is a local-debugging thing, but I also assume it'll be useful going forward, so lgtm?

Reply:
Yeah, this is useful for checking code size without all the libc printf/streams stuff.